src/heliaServer.ts

```diff
@@ -59,23 +59,36 @@ export class HeliaServer {
     ]
   }
 
+  /**
+   * Computes referer path for the request.
+   */
+  private fetchRefererPath ({ request }: IRouteHandler): string {
```

Comment:
```suggestion
  private getRefererFromRouteHandler ({ request }: IRouteHandler): string {
```
I don't think we should use the term `fetch` here.

Also, (joke) just because HTTP spec spells referrer wrong doesn't mean we need to? =P